op-devstack/dsl/safedb.go

```diff
@@ -9,17 +11,21 @@ type safeHeadDBProvider interface {
 	safeHeadAtL1Block(l1BlockNum uint64) *eth.SafeHeadResponse
 }
 
-func checkSafeHeadConsistent(t testreq.TestingT, maxL1BlockNum uint64, checkNode, sourceOfTruth safeHeadDBProvider) {
+func checkSafeHeadConsistent(t testreq.TestingT, maxL1BlockNum uint64, checkNode, sourceOfTruth safeHeadDBProvider, minRequiredL2Block *uint64) {
 	require := testreq.New(t)
 	l1BlockNum := maxL1BlockNum
-	matchedSomething := false
+	var minL2BlockRecorded *uint64
 	for {
-
 		actual := checkNode.safeHeadAtL1Block(l1BlockNum)
 		if actual == nil {
 			// No further safe head data available
 			// Stop iterating as long as we found _some_ data
-			require.Truef(matchedSomething, "no safe head data available at L1 block %v", l1BlockNum)
+			require.NotNil(minL2BlockRecorded, "no safe head data available at L1 block %v", l1BlockNum)
+			fmt.Printf("Require min block %v and got back to %v\n", minRequiredL2Block, minL2BlockRecorded)
```

Comment:
Did you mean to leave this print statement in?

Reply:
Nope that should have been removed. Good spot thank you.